docs/source/user/installation.rst

```diff
@@ -0,0 +1,105 @@
+.. _numba-cuda-installation:
+
+============
+Installation
+============
+
+Requirements
+============
+
+Supported GPUs
+--------------
+
+Numba supports all NVIDIA GPUs that are supported by the CUDA Toolkit it uses.
+Presently for CUDA 11 this ranges from Compute Capabilities 3.5 to 9.0, and for
+CUDA 12 this ranges from 5.0 to 12.1, depending on the exact installed version.
+
+
+Supported CUDA Toolkits
+-----------------------
+
+Numba-CUDA aims to support all minor versions of the two most recent CUDA
+Toolkit releases. Presently 11 and 12 are supported; CUDA 11.2 is the minimum
+required, because older releases (11.0 and 11.1) have a version of NVVM based on
+a previous and incompatible LLVM version.
+
+For further information about version compatibility between toolkit and driver
+versions, refer to :ref:`minor-version-compatibility`.
+
+
+Installation with a Python package manager
+==========================================
+
+Conda users can install the CUDA Toolkit into a conda environment.
+
+For CUDA 12::
+
+    $ conda install -c conda-forge numba-cuda "cuda-version>=12.0"
+
+Alternatively, you can install all CUDA 12 dependencies from PyPI via ``pip``::
+
+    $ pip install numba-cuda[cu12]
+
+For CUDA 11, ``cudatoolkit`` is required::
+
+    $ conda install -c conda-forge numba-cuda cudatoolkit "cuda-version>=11.2,<12.0"
```

Comment:
`cudatoolkit` is already implicitly added from the `cuda-bindings` dependency we added.
```suggestion
    $ conda install -c conda-forge numba-cuda "cuda-version>=11.2,<12.0"
```